teams/types.toml

```diff
@@ -39,3 +39,7 @@ extra-people = ["eholk"]
 
 [permissions]
 bors.chalk.review = true
+bors.rust.try = true
+perf = true
+crater = true
```

Comment:
Crater runs don't happen super often. Given that they take a large amount of expensive machinery to run it seems reasonable to limit its use. Is there any reason the types team members can't reach out to a compiler team member when this is needed?

Reply:
> Crater runs don't happen super often

Types team members run crater far more frequently than most other compiler contributors.

Reply:
Sounds find to me. cc @Mark-Simulacrum and @pietroalbini

Reply:
I'm fine with types having crater access.